docs/spec/spec.md

```diff
@@ -1,104 +1,414 @@
-# Specification: Ballerina Soap Library
+# Specification: Ballerina SOAP Library
 
 _Owners_: @shafreenAnfar @MadhukaHarith92  
```

Comment:
We should add your name here.

Reply:
Fixed in [0582c3e](https://github.com/ballerina-platform/module-ballerina-soap/pull/117/commits/0582c3eb0e040ac6129a04c88fb6b873ee97c791)